src/background.ts

```diff
@@ -145,6 +145,11 @@ if (!fs.existsSync(vvppEngineDir)) {
   fs.mkdirSync(vvppEngineDir);
 }
 
+const exportEngineInfoFilename = path.join(
```

Comment:
あ、エンジン情報以外も書いていくと思うので、変数名わかりやすくするのどうでしょう。
あとFilenameは文字通りファイル名だと勘違いしやすそうなので、パスだとわかるようPathかなと。
とりあえずまあ･･･こうとか？
```suggestion
const runtimeInfoPath = path.join(
```